frontend/src/components/NavBar.scss

```diff
@@ -0,0 +1,17 @@
+header {
+  display: flex;
+  justify-content: center;
+
+  .navBar {
+    display: flex;
+    justify-content: space-around;
+    width: 75vw;
+    img {
+      transform: scale(0.95);
```

Comment:
Still another recommendation, but for consistency's sake, you should use similar values for all these transforms (I've seen that somewhere, haven't I ? :p )
Here you use a 0.95 factor but in another file you use a 0.90 one IIRC